packages/jaeger-ui/src/components/common/ExternalLinks.test.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
:flushed:

Reply:
if this is a brand new file/code, the copyright should be The Jaeger Authors

Reply:
@yurishkuro when did this change? should any existing files be updated from "Uber Technologies, Inc." to "The Jaeger Authors"?
Also, the [CONTRIBUTING.md#license](https://github.com/jaegertracing/jaeger-ui/blob/master/CONTRIBUTING.md#license) should probably mention to use the current year, right?

Reply:
It's always been the guideline. The link also says Jaeger Authors.

Yes, we might fix the year there so that people can copy & paste, although a preferred way is to auto-generate the headers like we do in the other repos (e.g. `make fmt` adds them).